src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { nowUOrange } from '../../assets';
+import {
+    topNavLinks,
+    bottomNavLinks,
+    mobileNavLinks,
+} from '../../utils/constants';
+import NavLinkSection from './navLinkSection/NavLinkSection';
+import NavDropdown from './navDropdown/NavDropdown';
+import classes from './Navbar.module.scss';
+
+// import NewsletterSignupModal from '../newsletter-signup-modal/NewsletterSignupModal';
+
+const Navbar = ({ newsModal }) => {
+    const [dropdownNavDisplay, setDropdownNavDisplay] = useState(false);
+    const openDropdownNav = () => setDropdownNavDisplay(true);
+    const closeDropdownNav = () => setDropdownNavDisplay(false);
+
+    const [showNewsModal, setShowNewsModal] = useState(false);
+    const toggleNewsModal = () => {
+        setShowNewsModal((state) => !state);
+        setDropdownNavDisplay(false);
+    };
+
+    useEffect(() => {
+        if (showNewsModal) {
+            newsModal(true);
+            toggleNewsModal();
+        }
+    }, [showNewsModal, newsModal]);
+
+    return (
+        <header className={classes.navContainer}>
+            <nav className={classes.topNavbar}>
+                <ul className={classes.linkContainer}>
+                    <NavLinkSection
+                        navLink={topNavLinks}
+                        toggleNewsModal={toggleNewsModal}
+                    />
+                </ul>
+            </nav>
+            <nav className={classes.bottomNavbar}>
+                <Link to={'/'} className={classes.logoContainer}>
+                    <img className={classes.logo} src={nowUOrange} alt='logo' />
+                </Link>
+                <ul className={classes.linkContainer}>
+                    <NavLinkSection navLink={bottomNavLinks} />
+                </ul>
+                <div
+                    className={classes.navBurgerContainer}
+                    onMouseDown={openDropdownNav}
+                >
+                    <div className={classes.burger} />
```

Comment:
I haven't looked at this running but we use material icons in our codebase and there is a hamburger menu icon https://material.io/resources/icons/?icon=menu&style=baseline.

If you add the style:
```
&.xxsmall {
    font-size:4px;
  }
```
to `.material-icons`  in `App.scss` you could do the following instead of : `<div className={classes.burger} />`
```
<i className={classNames('xxsmall material-icons', classes.burger)}>menu</i>
```

Reply:
I see, thank you for showing me what this was. I will use the same thing for the X then.
This is a little over this pull request, but I hear icon fonts are bad for screen readers, which I feel like our app has a higher chance to serve people with disabilities. I feel like people with disabilities are more inclined to help others with similar disabilities. 
Either way, I will give this a shot.

Reply:
This would need to a be a sitewide decision so I'd rather follow the current standard instead of re-inventing the wheel but good thinking about screen readers (I wasn't aware)